app/models/authority.rb

```diff
@@ -35,7 +35,7 @@ def full_name_and_state
   end
 
   def covered?
-    !morph_name.blank?
+    !morph_name.blank? || !superceded_by.blank?
```

Comment:
We could change these from negation to be `.present?` to make it easier to read.